recipes/link_traits/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
```

Comment:
Please use jinja template variables as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L4-L6